### DISCUSSION TITLE
十月更新 - 2

### DISCUSSION THREAD

**Discussion by Richasy** [Richasy/Bili.Uwp#497] [通知]
## 更改记录

- 修复帮助\&支持页面的UI问题 @Richasy (#494)

## 🚀 功能

- 支持直播刷新 @Richasy (#493)
- 支持设置弹幕速度 @Richasy (#490)
- 评论添加UP标识 @Richasy (#487)
- 添加弹幕显示区域设置 @Richasy (#485)
- 添加跳转历史播放进度提示 (#359) @artiga033 (#451)

## 🐛 Bug 修复

- 移除高级弹幕 @Richasy (#491)
- 修复三连中止误点赞的问题 @Richasy (#484)
- 修复直播断连的问题 @Richasy (#465)

<hr /><em>This discussion was created from the release <a href='https://github.com/Richasy/Bili.Uwp/releases/tag/v2.2110.3.0'>十月更新 - 2</a>.</em>

**Comment by zxz4514** [comment]
大佬辛苦了

**Reply by ELI2sdkm** [reply]
感谢大佬

**Comment by chenzhenqi1001** [comment]
全屏播放卡顿，win11，5800X，6800XT。窗口化播放不卡顿，只有全屏的时候卡顿
